stdout of test 'pithole` in directory 'monetdb5/mal` itself:


# 14:45:52 >  
# 14:45:52 >   mserver5 "--config=/ufs/mk/monet5stable//Linux/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/ufs/mk/monet5stable//Linux/lib/MonetDB5:/ufs/mk/monet5stable//Linux/lib/MonetDB5/lib:/ufs/mk/monet5stable//Linux/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/mk/monet5stable//Linux/var/MonetDB5/dbfarm"  --set mapi_open=true --set xrpc_open=true --set mapi_port=30041 --set xrpc_port=49984 --set monet_prompt= --trace  --dbname=mTests_src_mal  pithole.mal
# 14:45:52 >  

# MonetDB server v5.14.0, based on kernel v1.32.0
# Serving database 'mTests_src_mal', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2009 MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://eir.ins.cwi.nl:30041/
function user.main():void;
#challenge the runtime stack protection 
    user.main();
end main;
!MALException:mal.interpreter:Recursive call limit reached.

# 14:45:52 >  
# 14:45:52 >  Done.
# 14:45:52 >  

